Nos/Views/NIP05View.swift

```diff
@@ -43,13 +43,17 @@ struct NIP05View: View {
                             username: nip05Identifier,
                             publicKey: publicKey
                         )
+                    } catch URLError.cancelled {
+                        // Cancelled requests are common in bad networks. We
+                        // should keep things as they were to avoid reproducing
+                        // https://github.com/planetary-social/nos/issues/1161
+                        return
                     } catch {
                         isVerified = false
-                        Log.debug(error.localizedDescription)
-                    }
-                    withAnimation {
-                        self.verifiedNip05Identifier = isVerified
+                        let message = error.localizedDescription
+                        Log.debug("Error while verifying a NIP-05: \(message)")
                     }
+                    self.verifiedNip05Identifier = isVerified
```

Comment:
I removed the animation because for the normal use case (a valid NIP-05), the animation causes an annoying blink. See relevant comment here https://planetary-app.slack.com/archives/CM4EPK324/p1717187450297749

I tried to add an `if` that executes that just when isVerified is different from verifiedNip05Identifier but it didn't work right away and decided not to continue with that line just because I think the animation is not important in this case and we can just remove it for all cases. If it were important, I think we should re-try the `if` line, eventually we will find out the solution.

Reply:
Makes sense to me.